source/chapters/library.rst

```diff
@@ -866,6 +867,86 @@ different location. Select :guilabel:`Use Default Library` to reset.
   Mac partition mounted in Linux, you can load it and use your iTunes tracks and
   playlists as well.
 
+.. _import-serato-metadata:
+
+Importing metadata from Serato
+------------------------------
```

Comment:
The section more than just metadata tags.

What do think about this?

```suggestion
.. _library-serato:

Using the Serato library
------------------------
```

Reply:
Sure! Since the section talks of more than just metadata tags

Reply:
By the way, you can apply apply these suggestions directly from GitHub web interface: https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

Reply:
> By the way, you can apply apply these suggestions directly from GitHub web interface: https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

Thank you so much! I would read this up and implement in future suggestions. Thank you so much!